### DISCUSSION TITLE
How to delete remote `spr/` branches after merge

### DISCUSSION THREAD

**Discussion by levibostian** [ejoffe/spr#381] [Q&A] [answered]
# Expected behavior

* Create a new branch off `main`. 
* Create commits on new branch. 
* Run `git spr update` to create stack. 
* Run `git spr merge` when ready to merge stack. 
* All open PRs get merged/closed by spr. Commits get merged into `main`. *And* all `spr/` branches on my remote repository that were created by `spr` to create the PR stack all get deleted. 

# Actual behavior 

* ...
* My github repo after running `git spr merge` will still have `spr/` branches that did not get deleted. I need to manually go into github and delete those branches. I have noticed that the branches that do not get deleted are branches that are associated with the PRs that got closed (not the one that got merged by `spr`). 

I am assuming I have something configured incorrectly? Maybe this is a feature request for `spr`? 

Any help is appreciated 😄

**Comment by ejoffe** [comment, marked as answer]
Automatic cleanup of merged branches is currently not implemented. Go ahead and open an issue to track this.

**Reply by levibostian** [reply]
I'll open one. Thanks for responding. 

Couple questions about the feature
- should this automatic cleanup be enabled by default?
- the idea behind this feature is that on `spr merge`, delete all remote branches that map to all of the pull requests that were closed. Correct?
